docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,162 @@
+
+- **default**: `true`
+- **description**: Enables or disables the warning when legacy formatting is detected.
+
+#### Paper.DisableClassPrioritization:
+
+- **default**: `unset`
+- **description**: Disables the class prioritization system - mostly an issue when failing to relocate or shade properly.
+
+#### Paper.disableFlushConsolidate:
+
+- **default**: `unset`
+- **description**: Disables the netty flush consolidation system.
+
+#### Paper.debugDynamicMissingKeys:
+
+- **default**: `unset`
+- **description**: Enables debug logging for missing keys in NBT objects.
+
+#### disable.watchdog:
+
+- **default**: `unset`
+- **description**: Disables the watchdog warning system.
+
+#### paper.explicit-flush:
+
+- **default**: `unset`
+- **description**: Enables explicit flushing of the network channel.
+
+#### Paper.enable-sync-chunk-writes:
+
+- **default**: `unset`
+- **description**: Syncs writes on each write call. This has a performance impact, particularly on hard drives.
+
+#### paper.debug-sync-loads:
+
+- **default**: `unset`
+- **description**: Enables debug logging for sync chunk loads.
+
+#### Paper.ignoreWorldDataVersion:
+
+- **default**: `unset`
+- **description**: Ignores the world data version when loading a world. This is not recommended and will likely cause issues.
+
+#### Paper.allowAsyncChunksSingleCore:
+
+- **default**: `unset`
+- **description**: Allows async chunk loading on single core systems.
+
+#### debug.entities:
+
+- **default**: `unset`
+- **description**: Enables debug log messages for entity information.
+
+#### Paper.bypassHostCheck:
+
+- **default**: `unset`
+- **description**: Bypasses the host pattern matching attempt for the client when connecting to the server.
+
+#### paper.ticklist-warn-on-excessive-delay:
+
+- **default**: `unset`
+- **description**: Enables the warning when a tick list is scheduled with an excessive delay.
+
+#### debug.rewriteForIde:
+
+- **default**: `unset`
+- **description**: Removes the NMS revision from the stack trace to allow for easier debugging in IDEs.
+
+#### convertLegacySigns:
+
+- **default**: `unset`
+- **description**: Converts legacy signs to the new format.
+
+#### paper.maxCustomChannelName:
+
+- **default**: `64`
+- **description**: Sets the largest size that a plugin channel can be.
+
+#### Paper.maxSignLength:
+
+- **default**: `80`
+- **description**: Sets the maximum line length for signs.
+
+#### Paper.FilterThreshhold:
+
+- **default**: `8192`
+- **min**: `8192`
+- **description**: Sets the maximum size of a packet to be filtered.
+
+#### Paper.minPrecachedDatafixVersion:
+
+- **default**: `Minecraft world version + 1`
+- **description**: If you are expecting to convert a large number of chunks you might consider setting this to only convert from a point onwards.
+
+#### paper.maxChunkThreads:
+
+- **default**: `4`
+- **description**: Sets the maximum number of threads to use for chunk loading. You cannot use more than half of the available cores.
```

Comment:
This flag no longer exists since the introduction of the new chunk system.

Reply:
> The ones removed in the last two versions I have kept

As it was a recent change I left them. Should I change this?

Reply:
Might as well remove them already, yea

Reply:
Ah, didn't see that. I don't think it makes sense to keep removed flags in this reference page though, since we have versioned docs.